TypeGap/TypeFluent.cs

```diff
@@ -30,6 +30,7 @@ public class TypeFluent
         private GapEnumGenerator _enumGenerator = new RegularEnumGenerator(false, EnumValueMode.Number);
         private string _indent = "    ";
         private ITsModelVisitor _modelVisitor;
+        private Dictionary<Type, string> _typeConversions;
```

Comment:
These changes shouldn't be in the PR, can you pull all commits from master and rebase your PR please?

Reply:
I will revert those